### PR TITLE
Add a spinner and a few tests

### PR DESCRIPTION
**Pull request by ekmartin**
I took the liberty of upgrading a few dependencies as well (some stuff had changed in xo, so fixed that).

Nifty little tool, way faster than checking imdb.com 👍 😎

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-const helpers = require('./helpers');
```
Comment (ekmartin) [inline src/api.js:3]: These changes were required by xo 0.16.0.

**Comment by ekmartin** [comment]
Also added a `.catch`-handler to the main API call, to print an error and stop the spinner.

**Comment by fernahh** [comment]
WOW! Nice improvements!

Thanks, @ekmartin! :beers:

**Comment by ekmartin** [comment]
Thanks! Would be cool if you released a new version :rocket:

**Comment by fernahh** [comment]
@ekmartin done! https://github.com/fernahh/imdbtr/releases/tag/v0.4.0

**Comment by ekmartin** [comment]
Cool! 👍

**Pull request merged**